### PR TITLE
Link GLMix paper to README.md and add more explanations to GAME.

### PR DESCRIPTION
**Pull request by XianXing**
Related issue: #116.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,15 +72,17 @@ In addition to the Generalized Linear Models described above, Photon-ML also sup
...
 GAME is a specific expansion of traditional Generalized Linear Models that further provides entity level (e.g., per-user/per-item) or segment level (e.g., per-country/per-category) coefficients, also known as random effects in the statistics literature, in addition to global coefficients. It manages to scale model training up to hundreds of billions of coefficients while still solvable within Spark's framework.
 
-GAME models consist of three components:
-  * One fixed effect model:
-    * The fixed effect model is effectively a conventional generalized linear model. Its parameters are "global" in the sense that they apply uniformly to all entities.
-  * Multiple random effect models:
-    * Random effect models consist of "local" parameters – entity-specific coefficients that can be seen as random deviations from the global mean. In other words, they are personalized models.
-  * Optionally a matrix factorization model:
-    * The matrix factorization model captures interactions between the different random effect models.
+Currently Photon-ML supports GAME models consist of the following three types of components:
```
Comment (ashelkovnykov) [inline README.md]: TYPO: consist --> composed

File: README.md [hunk 2]
```diff
@@ -71,15 +72,17 @@ In addition to the Generalized Linear Models described above, Photon-ML also sup
...
 
-The main difference between a GAME model and a conventional linear model is that GAME includes per-entity sub-models for personalization. An entity can be thought of as a logical grouping of data around some object or person, say a member. In GAME, each entity has its own RandomEffect optimization problem, where the training data have been grouped and partitioned by entity.
+For example, a GAME model for movie recommendation can be formulated as fixed effect model + per-user random effect model + per-movie random effect model + user-movie matrix factorization model. More details on GAME models can be found [here](https://docs.google.com/presentation/d/1vHanpK3KLIVgdDIHYRehUeyb04Hc2AasbBHs4InVPSU).
+
+One exemplary type of GAME model supported in Photon-ML is [GLMix](https://github.com/linkedin/photon-ml#reference), which has been adopted to serve the machine learning components of every LinkedIn's core products including LinkedIn's jobs search and recommendation, news feed ranking, Ads CTR prediction and "People Also Viewed". More details on GLMix models can be found [here](https://docs.google.com/presentation/d/1tYoelUma9-MMYdteWYS31LqVeoyPEncxJRk-k57gj0A/edit?usp=sharing).
```
Comment (ashelkovnykov) [inline README.md]: " .. which has been adopted to serve the machine learning components of LinkedIn's core products, including: job search and recommendation, news ..."

File: README.md [hunk 3]
```diff
@@ -71,15 +72,17 @@ In addition to the Generalized Linear Models described above, Photon-ML also sup
...
-  * Multiple random effect models:
-    * Random effect models consist of "local" parameters – entity-specific coefficients that can be seen as random deviations from the global mean. In other words, they are personalized models.
-  * Optionally a matrix factorization model:
-    * The matrix factorization model captures interactions between the different random effect models.
+Currently Photon-ML supports GAME models consist of the following three types of components:
+  * Fixed effect model:
+    * Each fixed effect model is effectively a conventional generalized linear model. Its parameters are "global" in the sense that they apply uniformly to all entities.
+  * Random effect model:
+    * Each random effect model consists of "local" parameters – entity-specific coefficients that can be seen as random deviations from the global mean, for example, a per-user random effect model that models each user's behavior through user-specific coefficients in additional to the fixed-effect coefficients shared by all users.
```
Comment (ashelkovnykov) [inline README.md]: "... from the global mean. For example, a per-user random effect models each user's behavior through user-specific coefficients."

I can't think of a way to add the statement about 'in addition to shared coefficients' without it sounding like a random effect model computes both. I think leaving the above + the links to the tutorial/presentation/paper should be quite clear.

**Comment by ashelkovnykov** [comment]
Don't let the big red X scare you, if I understood GitHub correctly you can still merge at any time

**Comment by XianXing** [comment]
At first glance, I thought my changes to README.md failed some grammatical or typo based unit tests.

**Comment by ashelkovnykov** [comment]
Will merge after Travis completes

**Pull request merged**